### DISCUSSION TITLE
Provide atomic operators to update queries

### DISCUSSION THREAD

**Discussion by tyteen4a03** [payloadcms/payload#9041] [Feature Requests & Ideas]
It would be great for Payload to provide atomic operators (like [`$inc` in MongoDB](https://www.mongodb.com/docs/manual/reference/operator/update/inc/)) for single-operation atomic updates, which allows the developer to skip opening and closing transactions to update a single field.

**Comment by DanRibbens** [comment]
That might work for simple scenarios. I'm not sure how feasible it is when you have validation, field hooks, access control or versions enabled. We could likely add support but it would require a transaction and the rest of the overhead of making an update.

I think the best you can do is to write your own using the mongoose models or custom SQL from a custom endpoint.
